eras/babbage/test-suite/src/Test/Cardano/Ledger/Babbage/Examples/Consensus.hs

```diff
@@ -52,13 +50,12 @@ import qualified Data.Map.Strict as Map
 import Data.Proxy (Proxy (..))
 import qualified Data.Sequence.Strict as StrictSeq
 import qualified Data.Set as Set
+import Lens.Micro
```

Comment:
is this a library that we also want to allow open imports for? :thinking:

Reply:
Yeah, for sure, since not it became omnipresent.